test/startsWith.test.js

```diff
@@ -0,0 +1,18 @@
+import test from 'ava'
+import {startsWith} from '../src'
+
+test('check correct input', t => {
+  const string = 'First PR'
+  const head = 'Fir'
+  const actual = startsWith(string, head)
+  const expected = true;
+  t.deepEqual(actual, expected)
+})
+
+test('check wrong input', t => {
+  const string = 'First PR'
+  const head = 'Foo'
+  const actual = startsWith(string, head)
+  const expected = false;
+  t.deepEqual(actual, expected)
```

Comment:
I was wandering - I made the required changes but I don't want to make another commit. I figured I'll amend my changes to the last commit but Iv'e seen in several sources a warning against amending code to published commits. Can you give a brief explanation on this issue? It seems I can amend my code because my PR hasn't been approved, thus my commit isn't published.